Sources/Networking/HTTPClient/HTTPRequestPath.swift

```diff
@@ -94,6 +96,17 @@ extension HTTPRequest.Path: HTTPRequestPath {
     // swiftlint:disable:next force_unwrapping
     static let serverHostURL = URL(string: "https://api.revenuecat.com")!
 
+    var fallbackPaths: [HTTPRequestPath] {
+        switch self {
+        case .getOfferings:
+            return [HTTPRequest.FallbackPath.getOfferings]
+        case .getProductEntitlementMapping:
+            return [HTTPRequest.FallbackPath.getProductEntitlementMapping]
```

Comment:
So we're only adding fallbacks for these 2 endpoints correct?

Reply:
That's right. Our fallback backend only supports these two endpoints at the moment.